toptal/picasso
# fix(FieldWrapper): don't pass the disabled property for label

[FX-2145](https://toptal-core.atlassian.net/browse/FX-2145)

### Description

Labels for inputs (with exception of checkboxes, radio buttons and switches) should not show a disabled label

### How to test

- Go to storybook and see the new example for disabled states, all labels with exception to radio, checkboxes and switches should be the same color as non-disabled ones

### Screenshots

| Before | After  |
| --- | -------- |
![story_picasso-forms-form--form-fx-2145-before](https://user-images.githubusercontent.com/5845160/158693091-ed408a66-a06d-482e-82a3-9ef27948ee0e.png) | ![story_picasso-forms-form--form-fx-2145-after](https://user-images.githubusercontent.com/5845160/158693105-88fd5606-0029-4091-a7f4-525bec42d8f1.png)

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>
